python/src/LeastSquaresExpansion_doc.i.in

```diff
@@ -0,0 +1,62 @@
+%feature("docstring") OT::LeastSquaresExpansion
+"L2 approximation on an orthonormal basis using least-squares and a fixed basis.
+
+Available constructors:
+    LeastSquaresExpansion(*inputSample, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, outputSample, distribution, basis, basisSize, methodName*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution, basis, basisSize, methodName*)
+
+Parameters
+----------
+inputSample : 2-d sequence of float of size :math:`n>0`
+    Input sample.
+weight : sequence of float of size :math:`n>0`, optional
+    Input weights. If not given, the default value is a uniform weight equals to :math:`1/n`.
+outputSample : 2-d sequence of floats of size :math:`n>0`
+    Output sample.
+distribution : :class:`~openturns.Distribution`
+    The distribution associated to the weighted sample *(inputSample, weights)*
+basis : :class:`~openturns.OrthogonalBasis`, optional
+    The orthonormal basis of the vector space on which the approximation is built.
+    If not given, a basis is built using an hyperbolic enumerate function
+    :class:`~openturns.HyperbolicAnisotropicEnumerateFunction` associated to the
+    :math:`q-`norm specified by the 'FunctionalChaosAlgorithm-QNorm' key in
```

Comment:
I do not known why, but this formats poorly:
![image](https://user-images.githubusercontent.com/31351465/235108888-0302a561-1a7a-411f-a6d4-8591ab8a2aaa.png)